src/project/FileTreeView.js

```diff
@@ -109,6 +111,40 @@ define(function (require, exports, module) {
     }
 
     /**
+     * @private
+     *
+     * Create an appropriate div based thickness to indent the tree correctly.
+     *
+     * @param {int} depth - The depthness of the current node.
+     * @return {ReactComponent} - The resulting div.
+     */
+    function _createThickness(depth) {
+        return DOM.div({
```

Comment:
I just saw it is defined in the `thickness` class, may be we can just define it inside the `style` prop?